eos-update-flatpak-repos

```diff
@@ -1418,7 +1418,7 @@ def _migrate_installed_flatpaks():
                                                  new_ostree_ref.split(':')[1],
                                                  **kwargs)
                         repo.commit_transaction()
-                    except:
+                    except Exception:
```

Comment:
Well, at the OS level the lock is just a lock on a file, and it's released when the process terminates. When `ostree_repo_prepare_transaction()` is run in some other process it recovers. (This has to work because otherwise a segfault in the body of the `try:` would leave the repo in a stuck state.

However, repo.abort_transaction() is a no-op if we're not in a transaction, so we can actually replace this `except` block with a `finally` block. I'll make that change.